Library/Formula/slimerjs.rb

```diff
@@ -1,3 +1,23 @@
+class FirefoxDependency < Requirement
+  fatal true
+  cask "Firefox"
+  download "https://www.mozilla.org/en-US/firefox/new"
+
+  satisfy { firefoxappdira? || firefoxappdirb? }
```

Comment:
That works for me. Thanks for the code block, always appreciated.

I presume we should be doing ` satisfy { firefox_installation } ` rather than ` satisfy { firefox_installation.nil? } ` otherwise it would fail *if* a Firefox installation *is* detected, rather than when it isn't?